src/applications/simple-forms/21-0972/pages/veteranIdentificationInformation2.js

```diff
@@ -20,8 +18,14 @@ export default {
     type: 'object',
     properties: {
       veteranSsn: ssnSchema,
-      veteranVaFileNumber: vaFileNumberSchema,
-      veteranServiceNumber: serviceNumberSchema,
+      veteranVaFileNumber: {
```

Comment:
does the source / web component need to be updated here instead of this?

Reply:
I went back to the shared definition and clarified the error message that the letters must be uppercase. I thought lowercase was allowed, so may have to check with design later on that.